Feat/como ser el mejor ing de tu equipo (#7)
bukhr/tech-blog: _posts/2024-01-22-2023-05-22-como-ser-el-mejor-dev-de-tu-equipo.md

---
layout: post

> title: Cómo ser el mejor dev de tu equipo

Reviewer: Podría ser algo así como:

4 pasos para ser el mejor ingeniero de tu equipo

Author: en realidad son 4 ejemplos concretos pero no es un step-by-step ... es solo un step:  "No lo hagas pesimo" === "contruye una red de confianza"

Reviewer: sí! en realidad me gustaría reformular el título a algo más pegote y porque a la gente le gusta leer listas o cosas que parezcan listas.

Author: voy con gpt a ver que se le ocurre... igual a mi me parecia que era pegote por ser bien "click bait"

> subtitle: Sobre cómo construir confianza

Reviewer: El secreto está en construir confianza.

Author: listoco

background: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo/portada.png"
images_path: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo"
author: Javier Vergara
tags: confianza desarrollo

> date: 2024-01-22 16:28 +0100

Reviewer: si pudieras poner una fecha más reciente, sería ideal

Author: listoco

---

> layout: post
> title: Cómo ser el mejor dev de tu equipo
> subtitle: Sobre cómo contruir confianza
> background: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo/portada.png"
> images_path: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo"
> author: Javier Vergara
> tags: confianza desarrollo
> date: 2024-01-22 16:28 +0100
> ---

Reviewer: ```suggestion
layout: post
title: Cómo ser el mejor dev de tu equipo
subtitle: Sobre cómo construir confianza
background: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo/portada.png"
images_path: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo"
author: Javier Vergara
tags: confianza desarrollo
date: 2024-01-22 16:28 +0100
---
```

> Bajo un título que huele a clickbait espero poder entregarte un consejo que de verdad te ayude en tu carrera como ingeniero de software

Reviewer: ```suggestion
Quizás el título huele a clickbait, pero espero poder entregarte un consejo que sí te ayude en tu carrera como ingeniero de software.
```

> No se si te pasa como a mi, que estoy un poco aburrido de tanto consejo “esotérico” pululando por ahí: rutinas avaladas por neurólogos sobre cuál es la hora óptima de dormir o despertar,
> millonarios aconsejando trabajar 12 horas por 7 días a la semana, monstruosos toolset de herramientas para tomar notas y la manera correcta de organizarlas, u otro framework “agile” de moda. 

Reviewer: ```suggestion
No sé si te pasa como a mí, que estoy un poco aburrido de tanto consejo _esotérico_ pululando por ahí: rutinas avaladas por neurólogos sobre cuál es la hora óptima de dormir o despertar, millonarios aconsejando trabajar 12 horas por 7 días a la semana, monstruosos toolsets de herramientas para tomar notas y la manera correcta de organizarlas, u otro framework “agile” de moda. 
```
Quizás podrías aquí agregar el link de alguno de esos artículos que te generan desconfianza

Author: sorry ni tengo los links, pero yo diría que a cada uno le hace eco a mas o menos que me refiero... y no quiero ofender tampoco xD

> Por estas razones, me sorprendió encontrarme hace unas semanas con una conferencia bajo el título “Grow Your Own Tech Leads • Ken Scambler • YOW! 2019” en YT, 
> la cual se aleja totalmente de de los consejos esotéricos y vuelve a algo completamente básico y esencial en los equipos, la confianza.

Reviewer: ```suggestion
Por estas razones, me sorprendió encontrarme hace unas semanas con una conferencia titulada **“Grow Your Own Tech Leads • Ken Scambler • YOW! 2019”** en Youtube, que se aleja totalmente de de los consejos esotéricos y vuelve a algo completamente básico y esencial en los equipos: **la confianza.**
```
Link a la charla!

Author: lo había dejado al final del post, para que no se desconcentren y terminen en YT en vez de leer mi post xD... lo mantengo abajo?

> Este artículo no es más que un modesto resumen de la conferencia que mencioné (link al final), y en honor al tiempo que tomaste en leer esta entrada,
> te dejo de inmediato con el consejo que te va a convertir en un ingeniero de otro planeta, redoble de tambores…

Reviewer: ```suggestion
Es por esto que quiero presentarte un modesto resumen de la conferencia, y en honor al tiempo que tomaste en leer esta entrada, te dejo de inmediato con el consejo que te convertirá en un ingeniero de otro planeta. Redoble de tambores…

```

>  > ## no lo hagas pésimo ##

Reviewer: ```suggestion
 > ## No lo hagas pésimo ##
```

> Medio en broma, medio en serio, así de simple, en inglés el consejo era “don't be terrible”. Ten un poco de paciencia y te explico con unos ejemplos a qué me refiero.

Reviewer: ```suggestion
Medio en broma, medio en serio.  Así de simple. En inglés el consejo es **“don't be terrible”**. Ten un poco de paciencia y te explico con unos ejemplos a qué me refiero.
```

### 1. El action item de la retro ###

> De seguro tras salir de la retrospectiva te anotaste para algun action item, habitualmente está en una actividad voluntaria, y tras llegar a la siguiente retro
> tu jefe te pregunta como te fue con el action item y… olvidaste hacerlo. Pesimo 😞
> 
> En estas tareas sencillas, es donde se construye confianza y muchos fallan en darle la importancia que merece, ya que están más preocupados en convertirse en el 10x programmer,
> memorizando cada patrón de diseño, estructura de datos y algoritmo en leetcode. 
> 
> El efecto que tiene esto en tu jefe es que tendrá que encontrar un espacio en su calendario que ya luce como un tetris
> 
> ![tetris_calendar]({{page.images_path}}/tetris-calendar.png)
> 
> preguntando constantemente el estado de avance de tu tarea, ya que perdiste confianza.  
> 
> Aquí tienes una primera señal, si tu jefe está pidiendo constantemente un status, puede que lo estés haciendo terrible.
> 
> Informa tu avance de manera proactiva, así no tienen que venir a molestarte.
> 

Reviewer: ```suggestion
De seguro tras salir de la retrospectiva te anotaste para algun action item, habitualmente está en una actividad voluntaria, y tras llegar a la siguiente retro, tu jefe te pregunta cómo te fue con el action item y… olvidaste hacerlo. Pésimo 😞

En estas tareas sencillas es donde se construye confianza y muchos fallan en darle la importancia que merece, ya que están más preocupados en convertirse en el **10x programmer**, memorizando cada patrón de diseño, estructura de datos y algoritmos en Leetcode. 

El efecto que tiene esto en tu jefe, es que tendrá que encontrar un espacio en su calendario que **ya luce como un tetris**, preguntando constantemente el estado de avance de tu tarea, ya que perdiste confianza.  

![tetris_calendar]({{page.images_path}}/tetris-calendar.png)

Aquí tienes una primera señal. Si tu jefe pide constantemente un status, puede que lo estés haciendo terrible.

**Informa tu avance de manera proactiva, así no tienen que molestarte.**

```

Link a leetcode!

Author: listoco
